### ISSUE TITLE
Accessibility: Screen readers can't read emoji reactions

### ISSUE THREAD

**Issue by farewelltospring** [signalapp/Signal-Android#13358]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Screen readers can't read emoji reactions. Selecting it in Talkback just says "unlabeled".

### Steps to reproduce
- Add reaction to message (cannot be done in talkback)
- Enable talkback
- Select reactions pill in talkback

**Actual result:** Talkback says "unlabeled"
**Expected result:** Talkback should say the name of the emoji

### Screenshots
![image](https://github.com/signalapp/Signal-Android/assets/108375980/aac0e4bb-e2af-4bce-af6f-5039cd81a956)

### Device info

**Device:** Pixel 3
**Android version:** 13
**Signal version:** 6.43.2

### Link to debug log
n/a
